test/helpers.test.ts

```diff
@@ -123,4 +128,53 @@ describe('helpers', () => {
       expect(() => shift(["a"])).not.toThrow();
     });
   });
+
+  describe('pkce', () => {
+    describe('code_verifier', () => {
+      it('should accept a valid PKCE code_verifier', () => {
+        const verifier128 =
+          'PXa7p8YHHUAJGrcG2eW0x7FY_EBtRTlaUHnyz1jKWnNp0G-2HZt9KjA0UOp87DmuIqoV4Y_owVsM-QICvrSa5dWxOndVEhSsFMMgy68AYkw4PGHkGaN_aIRIHJ8mQ4EZ';
+        const verifier42 = 'xyo94uhy3zKvgB0NJwLms86SwcjtWviEOpkBnGgaLlo';
+        expect(isValidPkceCodeVerifier(verifier128)).toBeTruthy();
+        expect(isValidPkceCodeVerifier(verifier42)).toBeTruthy();
+
+        const verifierWith129chars = `${verifier128}a`;
+        expect(isValidPkceCodeVerifier(verifierWith129chars)).toBeFalsy();
```

Comment:
Similarly could you please switch to `toBe(false)` here and everywhere else?

Reply:
Fixed!